Rdmp.UI/Properties/Resources.Designer.cs

```diff
@@ -0,0 +1,63 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
```

Comment:
Any idea where  `Rdmp.UI/Properties/Resources.Designer.cs` and `Resources.resx` came from? They seem to be empty